--- package.use.mask	2019-12-29 15:56:17.316895003 +0100
+++ package.use.mask.patched	2019-12-29 15:57:38.276039188 +0100
@@ -551,7 +551,10 @@
 
 # Chí-Thanh Christopher Nguyễn <chithanh@gentoo.org> (2013-07-13)
 # GL/GLES support in cairo is mutually exclusive, bug #428770.
-x11-libs/cairo gles2-only
+#x11-libs/cairo gles2-only
+
+#overriden by your gles2 choice, disabling the opengl backend instead
+x11-libs/cairo opengl
 
 # Kacper Kowalik <xarthisius@gentoo.org> (2013-05-19)
 # Fails to build, haven't had time to debug
