diff --git a/arch/arm64/Makefile b/arch/arm64/Makefile
index 2c0238ce0..37e952c08 100644
--- a/arch/arm64/Makefile
+++ b/arch/arm64/Makefile
@@ -14,6 +14,10 @@ LDFLAGS_vmlinux	:=--no-undefined -X
 CPPFLAGS_vmlinux.lds = -DTEXT_OFFSET=$(TEXT_OFFSET)
 GZFLAGS		:=-9
 
+ifeq ($(CONFIG_CC_A72.A53), y)
+KBUILD_CFLAGS += -mcpu=cortex-a72.cortex-a53
+endif
+
 ifeq ($(CONFIG_RELOCATABLE), y)
 # Pass --no-apply-dynamic-relocs to restore pre-binutils-2.27 behaviour
 # for relative relocs, since this leads to better Image compression
diff --git a/init/Kconfig b/init/Kconfig
index b4daad2ba..d37899ae7 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1243,8 +1243,7 @@ config CC_OPTIMIZE_FOR_PERFORMANCE
 	  helpful compile-time warnings.
 
 config CC_OPTIMIZE_FOR_PERFORMANCE_O3
-	bool "Optimize more for performance (-O3)"
-	depends on ARC
+	bool "UNTESTED!!! Optimize more for performance (-O3)"
 	imply CC_DISABLE_WARN_MAYBE_UNINITIALIZED  # avoid false positives
 	help
 	  Choosing this option will pass "-O3" to your compiler to optimize
@@ -1259,6 +1258,9 @@ config CC_OPTIMIZE_FOR_SIZE
 
 endchoice
 
+config CC_A72.A53
+	bool "pass -mcpu=cortex-a72.cortex-a53 to the compiler
+
 config HAVE_LD_DEAD_CODE_DATA_ELIMINATION
 	bool
 	help
